google/auth/transport/aiohttp_requests.py

```diff
@@ -31,6 +32,51 @@
 _DEFAULT_TIMEOUT = 180  # in seconds
 
 
+class _CombinedResponse(transport.Response):
+    '''
+    In order to more closely resemble the `requests` interface, where a raw
+    and deflated content could be accessed at once, this class lazily reads the 
+    stream in `transport.Response` so both return forms can be used
+    '''
+    def __init__(self, response):
+        self._response = response
+        self._raw_content = None
+
+    def _is_compressed(self):
+        # The gzip and deflate transfer-encodings are automatically decoded for you.
```

Comment:
This comment is a bit unclear, maybe rephrase.

Reply:
possibly move this into the comment up top as this is the point of this added class.

Reply:
Fleshed out this comment in the latest version to explain reasoning.